PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -28,15 +29,24 @@ struct StoreAppView: View {
                                 isList: isList,
                                 warningSymbol: $warningSymbol,
                                 warningMessage: $warningMessage)
-        .gesture(TapGesture(count: 2).onEnded {
```

Comment:
This gesture should be added back. Currently, the only way to download is through the context menu

Reply:
Is it ok to it just start download?